src/resources/js/components/AccountCreation.vue

```diff
@@ -0,0 +1,139 @@
+<template>
+  <v-container>
+    <h2 class="title">Inscription</h2>
+    <v-form
+        ref="form"
+        v-model="valid"
+        lazy-validation
+    >
+      <v-alert
+            v-model="message.show"
+            :class="message.type"
+            dismissible
+            >
+            {{ message.content }}
+        </v-alert>
+      <v-text-field
+            v-model="input.fname"
+            :rules="fnameRules"
+            required
+            label="Prénom"
+      />
+
+      <v-text-field
+            v-model="input.lname"
+            :rules="lnameRules"
+            required
+            label="Nom"
+      />
+
+      <v-text-field
+            v-model="input.email"
+            :rules="emailRules"
+            required
+            label="Adresse email"
+      />
+
+      <v-text-field
+            v-model="input.password"
+            :append-icon="showPassword ? 'mdi-eye' : 'mdi-eye-off'"
+            :type="showPassword ? 'text' : 'password'"
+            :rules="passwordRules"
+            @click:append="showPassword = !showPassword"
+            required
+            label="Mot de passe"
+      />
+
+      <v-text-field
+            v-model="input.passwordConfirm"
+            :type="showPassword ? 'text' : 'password'"
+            required
+            label="Confirmation de mot de passe"
+      />
```

Comment:
there should be a icon to show and hide password and a different value instead of ``showpassword``, now if you click on the eyes it show the both password,

Reply:
Changed in 9e44c92